components/org.wso2.carbon.identity.scim2.common/src/main/java/org/wso2/carbon/identity/scim2/common/impl/SCIMUserManager.java

```diff
@@ -309,6 +309,24 @@ public void deleteUser(String userId) throws NotFoundException, CharonException
         }
     }
 
+    @Override
+    public List<Object> listUsersWithGET(Node rootNode, Integer startIndex, Integer count, String sortBy,
+            String sortOrder, String domainName, Map<String, Boolean> requiredAttributes)
+            throws CharonException, NotImplementedException {
+
+        // Validate NULL value for startIndex.
+        startIndex = handleStartIndexEqualsNULL(startIndex);
```

Comment:
We already did the null checks in Charon under processStartIndex() method, so null won't be reached here, is there any reason to do it here again?

Reply:
This is a fix asked in a previous commit as a precaution.